### PR TITLE
f-content-cards@1.15.0: Add `Home_Promotion_Card_2` template

### PR DESCRIPTION
**Pull request by asherstoppard**
**Do Not Merge:** Awaiting #299 

## UI Review Checks

- [x] README and/or UI Documentation has been [created|updated]
- [x] Unit tests have been [created|updated]
- [x] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

### CODE REVIEW

**Comment by asherstoppard** [comment]
Manually ran build/lint/test and confirmed functionality whilst awaiting the unblocking of circleci

**Review by simonsandersje — APPROVED** [review]

File: packages/f-content-cards/src/components/cardTemplates/homePromotionCard/HomePromotionCard1.vue [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            type: Object,
+            default: () => ({})
+        },
+        containerMaxWidth: {
+            type: Number,
+            default: 1272
+        },
+        testId: {
+            type: String,
+            default: 'contentCard-homePromotionCard1'
```
Comment (minrwhite) [inline packages/f-content-cards/src/components/cardTemplates/homePromotionCard/HomePromotionCard1.vue]: Can we make this `null`y by default, please?

File: packages/f-content-cards/src/components/cardTemplates/homePromotionCard/HomePromotionCard2.vue [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+<script>
+export default {
+    props: {
+        card: {
+            type: Object,
+            default: () => ({})
+        },
+        testId: {
+            type: String,
+            default: 'contentCard-homePromotionCard2'
```
Comment (minrwhite) [inline packages/f-content-cards/src/components/cardTemplates/homePromotionCard/HomePromotionCard2.vue]: And here, please:

Can we make this `null`y by default, please?

**Review by minrwhite — COMMENTED** [review]
A couple comments - we also need to remove 'HomeContentCard.test.js', which refers to the now removed 'HomeContentCard.vue'.

**Comment by asherstoppard** [comment]
I've approved the f-content-cards screenshot but I'm raising a ticket to add header cards to each section in order to segregate each card type and make for more accurate checks

**Review by minrwhite — APPROVED** [review]

**Pull request merged**